controller/planner_backlog_test.go

```diff
@@ -79,13 +79,12 @@ func (rest *TestPlannerBacklogREST) setupPlannerBacklogWorkItems() (testSpace *s
 		require.Nil(rest.T(), err)
 		log.Info(nil, map[string]interface{}{"wit_id": workitemType.ID}, "created workitem type")
 
-		userActive := false
 		iterationsRepo := app.Iterations()
 		parentIteration = &iteration.Iteration{
 			Name:       "Parent Iteration",
 			SpaceID:    testSpace.ID,
 			State:      iteration.IterationStateNew,
-			UserActive: &userActive,
+			UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c